src/DotNetWorker.Core/Converters/Converter/DefaultConverterContext.cs

```diff
@@ -1,23 +1,48 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
-using Microsoft.Azure.Functions.Worker.Pipeline;
+using System;
+using System.Collections.Generic;
 
 namespace Microsoft.Azure.Functions.Worker.Converters
 {
-    internal class DefaultConverterContext : ConverterContext
+    /// <summary>
+    /// A type defining the information needed for an input conversion operation.
+    /// </summary>
+    public sealed class DefaultConverterContext : ConverterContext
     {
-        public DefaultConverterContext(FunctionParameter parameter, object? source, FunctionContext context)
+        /// <summary>
+        /// Creates a new instance of <see cref="DefaultConverterContext"/>
+        /// </summary>
+        /// <param name="targetType">The target type.</param>
+        /// <param name="source">The source.</param>
+        /// <param name="context">The function context.</param>
+        public DefaultConverterContext(Type targetType, object? source, FunctionContext context)
```

Comment:
nit: is it better practice to put the optional parameter at the end, or it doesn't matter?

Reply:
If you meant the `source` parameter, it is not an optional parameter. It is nullable, So the caller must provide a value(null or non null) for this parameter.